### ISSUE TITLE
Lucee 5?

### ISSUE THREAD

**Issue by MBlagui** [foundeo/cfdocs#341]

**Comment by pfreitag** [comment]
Yes - I have added some already, see: http://cfdocs.org/lucee5 for a list of what has been added so far.

**Comment by pfreitag** [comment]
There are some other new functions that need to be added: http://docs.lucee.org/guides/lucee-5/functions-tags.html as well
